docs/maintainers/faqs.md

```diff
@@ -131,6 +131,24 @@ Short answer: no. For the moment, we have all the v3 tracks in this single monor
 
 The basic communication channel for maintainers is Slack, and in particular the maintainer-only `#maintaining-<track>` channel. For communication with everyone else, GitHub issues should be used, for obvious reasons (discoverability, familiarity, etc.). We've also enabled the GitHub Projects feature on the v3 repo, so you can organize your track's v3 transition there too.
 
+## Should I use citations
+
+Yes! Please use citations in any reference documentation (files under `**/reference/**`). We are using the [Science of Computer Programming citation style][socp-citation-style]. This means that when citing website, please use this format:
+
+> $TITLE, $SOURCE. ($PUBLISHED_DATE). $URL (accessed \$ACCESSED_DATE).
+
+For example:
+
+> [1] Actor model, Wikipedia. (2020). https://en.wikipedia.org/w/index.php?title=Actor_model&oldid=939106706 (accessed February 29, 2020).
+
+To include a citation in reference documentation, add the citation reference to the bottom of the page. Then add a line with the `---` separator above the citation. The final step is to append a superscript to the cited text. This superscript should refer to the citation reference's number (usually `<sup>1</sup>`). See the [actor model reference documentation][actor_model-reference-documentation] for an example.
+
+Note that if the cited text is copied literally, please put it in [blockquotes][blockquotes].
+
+### Generating citations
+
+To help generate these citations, [ZoteroBib][zoterobib] can be used. On that website, first set the citation style to "Science of Computer Programming" in the "Bibliography" section. Then, paste the URL you want to cite in the input field at the top of the page, click on the "Cite" button and the citation will be generated for you.
```

Comment:
![Select citation format dropdown](https://user-images.githubusercontent.com/1964376/76363289-01988780-6323-11ea-861b-f9b7e074f540.png)


![Select citation format search box](https://user-images.githubusercontent.com/1964376/76363194-d1e97f80-6322-11ea-81fe-96caa35ca74e.png)

Maybe we want to include these?

Reply:
You mean the fact that you have to add it yourself?

Reply:
I think he's suggesting we add screenshots to the FAQ